src/doc/rustc/src/platform-support/fuchsia.md

```diff
@@ -687,7 +687,7 @@ Rust compiler locally. See "[Targeting Fuchsia with a compiler built from source
 for the steps to build locally.
 
 You'll also need to download a copy of the Fuchsia SDK. The current minimum
-supported SDK version is [9.20220726.1.1](https://chrome-infra-packages.appspot.com/p/fuchsia/sdk/core/linux-amd64/+/version:9.20220726.1.1).
+supported SDK version is [10.20221207.2.89][minimum_supported_sdk_version].
```

Comment:
This means you have to update two completely different areas of the doc when bumping the version, can you bring them back together? Using this style of link is fine, but I think it should be defined below this paragraph.